app/services/usps_in_person_proofing/enrollment_helper.rb

```diff
@@ -1,6 +1,6 @@
 module UspsInPersonProofing
   class EnrollmentHelper
-    def save_in_person_enrollment(user, profile, pii)
+    def save_in_person_enrollment(user, profile, pii, selected_location_details = nil)
```

Comment:
Maybe a later pull request if we're not yet targeting GPO support, but we need to figure out how to pass this from `GpoVerifyForm`:

https://github.com/18F/identity-idp/blob/1c84c38ae571568959dca22f28466e834120b10f/app/forms/gpo_verify_form.rb#L22-L26

This is difficult because (a) we can't rely on the session and (b) the enrollment record hasn't been created yet.

My initial instinct is that we should do something like what I described as the second option of https://github.com/18F/identity-idp/pull/6629#discussion_r932145855 : always create the enrollment in the initial flow, but keep it establishing and wait to send it to USPS. That way, we have the selected location details saved in the record. The challenge is that we need to update how we're thinking about deadline dates, since we cannot use creation date of the record (it may be different from the USPS enrollment creation).

Reply:
@tomas-nava Would you want to see that here, or should we follow-up with that as part of the GPO supports implementation?

Reply:
let's follow up with that